src/intervals/trigonometric.jl

```diff
@@ -137,6 +137,10 @@ function sin(a::Interval{Float64})
     end
 end
 
+function sinpi(a::Interval{T}) where T
+    w = *(a,pi_interval(T))
```

Comment:
Write this in the normal way:
`a * pi_interval(T)`.

In fact, it turns out that we can write `T(pi)` instead and remove `pi_interval`.

Reply:
By the way, when you use commas, add a space after the comma.

Reply:
pi_interval(T) and T(pi) give different results for the function. If we see that sinpi(a .. b) shall return the same value as sin(pi*a .. pi*b), then this hypothesis is satisfied by pi_interval.

Reply:
Yes sorry, I meant where `T` is the type of `a`.